### PR TITLE
Uglify the code

### PR DESCRIPTION
**Pull request by piotrbienias-dna**
- Removed semantic HTML and replaced with `div` elements,
- Product page now works (doesn't return 404 anymore),
- Refactored `FeaturedProducts` a little bit to make it a client component (space for improvement).

### CODE REVIEW

File: src/app/components/Navbar/index.tsx [hunk 1]
```diff
@@ -11,10 +10,10 @@ import Button from "../Button";
 
 export default function Navbar() {
   return (
-    <nav className={`${styles["navbar-wrapper"]} container`}>
+    <div className={`${styles["navbar-wrapper"]} container`}>
       <div className={styles.navbar}>
         <a href="localhost:3000">
-          <Image src="logo.svg" alt="Logo" width={94} height={35} />
+          <img src="logo.svg" alt="Logo" width={94} height={35} />
```
Comment (madecki) [inline src/app/components/Navbar/index.tsx:16]: Maybe we should get rid of size attributes, so this image will "jump" while loading?

File: src/pages/api/featured-products.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export default async function handler(
 ) {
   setTimeout(() => {
     res.send(products);
-  }, 1500);
+  }, 750);
```
Comment (madecki) [inline src/pages/api/featured-products.ts]: I added long loading to make him start thinking about adding a loader. He might not have time to even get to this part of website before it gets loaded.

Comment (piotrbienias-dna) [reply src/pages/api/featured-products.ts]: But I changed the `FeaturedProducts` to be purely client side, so now when we open the page there are no products and they showed up after 1,5 seconds, so I just decided to make it shorter, as looking at a blank section for that long was kinda annoying and suspicious :D I suggest running it locally to see what I mean and we can easily revert this change if we want :)

File: src/app/components/Navbar/index.tsx (hunk 1 above)
Comment (piotrbienias-dna) [reply src/app/components/Navbar/index.tsx:16]: Sure, I can try it locally to see how it behaves!

**Pull request merged**